firebase-admin/runtime/pom.xml

```diff
@@ -38,6 +39,19 @@ quarkus-google-cloud-common-grpc and Quarkus includes the netty library -->
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-http</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-security</artifactId>
+        </dependency>
```

Comment:
It is already a dependency of the `quarkus-google-cloud-common` module so it should not be needed

Reply:
Yes, I noticed, but I'm using QuarkusSecurityIdentity, which is in `io.quarkus.security.runtime`. It doesn't seem like it's getting through, though. I'm not sure if there's something special that needs to be done when using the runtime package.